### DISCUSSION TITLE
How attach my.svg icon from /assets to nz-icon?

### DISCUSSION THREAD

**Discussion by omqmo** [NG-ZORRO/ng-zorro-antd#7075] [General]
How attach my.svg icon from /assets to nz-icon?

**Comment by stygian-desolator** [comment]
try this https://ng.ant.design/components/icon/en#namespace

**Comment by jeena09** [comment]
To attach your custom `.svg` icon to `nz-icon`, you can do the following:

1. Import your SVG icon into your Angular project.
2. Use `NzIconService` to add the custom icon:
   ```typescript
   this.iconService.addIconFromUrl('assets/my.svg');
   ```
3. In your template, reference it using the icon name:
   ```html
   <i nz-icon [nzType]="'my'"></i>
   ```

This should display your custom SVG icon.
https://shayari4us.com/
